dandi/girder.py

```diff
@@ -375,7 +381,22 @@ def download_file(self, file_id, path, existing="error", attrs=None):
         #  in the progressbar label
         # For starters we would do this implementation but later RF
         # when RF - do not forget to remove progressReporterCls in __init__
-        self.downloadFile(file_id, path)
+
+        # Will do 3 attempts to avoid some problems due to flaky/overloaded
+        # connections, see https://github.com/dandi/dandi-cli/issues/87
+        for attempt in range(3):
+            try:
+                self.downloadFile(file_id, path)
+                break
+            except gcl.HttpError as exc:
```

Comment:
this piece of code is not even smoke tested so whoever finds a bug, get a cookie!